build.gradle

```diff
@@ -2832,7 +2824,7 @@ project(':streams:streams-scala') {
     dependsOn 'copyDependantLibs'
   }
 
-  // spotless 6.14 requires Java 11 at runtime
+  // spotless 6.25 requires Java 11 at runtime
   if (JavaVersion.current().isJava11Compatible()) {
```

Comment:
Can we also OR Java 17 in this if condition?

Reply:
this check is unnecessary since we assume the JDK 8 is dropped